long-read-rna-pipeline.wdl

```diff
@@ -38,6 +42,16 @@ workflow long_read_rna_pipeline {
         # of the run.
         Boolean canonical_only = true
         # Resouces
+        Resources concatenate_files_resources = {
+           "cpu": 2,
+           "memory_gb": 4,
+           "disks": "local-disk 50",
+        }
+        Resources make_gtf_from_spikein_fasta_resources = {
+           "cpu": 2,
+           "memory_gb": 4,
+           "disks": "local-disk 50",
+        }
```

Comment:
Would it make sense to have a generic resources struct somewhere else so you don't need to repeat the declaration? Just a though, I'm not sure how DNase does it. Not really a request for changes but an observation.

Reply:
Fair point. I am going to make the resource definitions cleaner in another branch later. I am not sure if I am going to go dnase way, or something else. The least I will do is I am going to define all the resources in an uniform way using structs.